### DISCUSSION TITLE
The tester toolkit 🧰

### DISCUSSION THREAD

**Discussion by Yehonal** [chromiecraft/chromiecraft#5] [Guides]
Here we're going to collect all the tools needed for a proper testing workflow

So first of all, prepare your tools

## The #test channel 💬

On our [Discord](https://discord.gg/BYtPEacERr) we have the #test channel that can be used to have a quick chat about certain feedbacks/test before creating a post or an issue on Github

## The feedback addon 🧰

[FeedbackUI Download](https://github.com/wowgaming/FeedbackUI/releases/latest)

Download or git-clone this addon the repo inside your /Interface/AddOns folder of the client directory. 

Follow the [README](https://github.com/wowgaming/FeedbackUI/blob/master/README.md) to understand how to use it

**Your in-game feedback will be shown on the website where you will be able to comment and vote them (Soon™)**

## The feedback discussion section ❤️

[HERE ](https://github.com/chromiecraft/chromiecraft/discussions/categories/feedbacks) you can discuss about suggestions, show your tests, but also leave positive feedback about something you did. It's basically the place where testers can discuss in a forum-like way.

## The bug-tracker 🐛

[THIS bugtracker](https://github.com/chromiecraft/chromiecraft/issues) is not intended to replace the official [AzerothCore bugtracker](https://github.com/azerothcore/azerothcore-wotlk/issues). every valid issue MUST be connected with an issue or a pull-request of the AzerothCore project.

Therefore, the bug tracker will be used for 2 reasons mainly:

* Allowing the casual-testers of the ChromieCraft project to have a board where easily post their bug-reports that will be then evaluated and moved to a proper AC issue/PR

* Having a visual representation of the progress made specifically by the ChromieCraft project subdivided in different progression brackets: https://github.com/chromiecraft/chromiecraft/projects

**Comment by locus313** [comment]
@Yehonal the FeedbackUI Download link is still pointing to the initial version of v1.2.5. You might want to point the link to https://github.com/wowgaming/FeedbackUI/releases/latest instead.

**Reply by Yehonal** [reply]
Thank you @locus313 !
